frontend/src/app/Components/MainPageSections/shorten-section/shorten-section.component.html

```diff
@@ -3,11 +3,52 @@
   <h1>BFFL</h1>
   <p>Welcome to BFFL - the famous URL-Shortener </p>
 
-  <form>
-    <label for="original">Original URL</label>
-    <input type="text" id="original" placeholder="Insert Original URL here">
-    <label for="wish">Short URL</label>
-    <input type="text" id = "wish" placeholder="Insert Wish for Short URL here">
-    <button type="submit">Gekürzte URL laden</button>
+  <form [formGroup]="shortenURLForm" (ngSubmit)="onSubmit()" id = "myform">
+    <label for="originalURL">Original URL</label>
+    <input type="text" id="originalURL" placeholder="Please insert your original URL here, for example 'https://mywebsite/impressum.com'" formControlName="originalURL" required/>
+    <label for="wishURL">What do you want as your short URL?</label>
+    <input type="text" id = "wishURL" placeholder="If you want 'https://bfflshort.de/group/impressum.de' as a short URL please insert 'impressum'" formControlName="wishURL" required/>
```

Comment:
Placeholders of line 8 and 10 seem very long to me, does it look appropriate in the application? (A placeholder should be a hint, not a full tutorial)